assets/src/containers/GradeDistribution.js

```diff
@@ -34,21 +34,30 @@ function GradeDistribution (props) {
   if (loaded && isObjectEmpty(gradeData)) return (<Error>No data provided.</Error>)
 
   const buildGradeView = gradeData => {
+    const grades = gradeData.map(x => x.current_grade)
     return (
       <Grid container>
         <Grid item xs={12} lg={2}>
           <Table className={classes.table} noBorder tableData={[
-            ['My Grade', <strong>{gradeData[0].current_user_grade
-              ? `${roundToOneDecimcal(gradeData[0].current_user_grade)}%`
-              : 'There are no grades yet for you in this course'}</strong>
+            [
+              'My Grade', <strong>{gradeData[0].current_user_grade
+                ? `${roundToOneDecimcal(gradeData[0].current_user_grade)}%`
+                : 'There are no grades yet for you in this course'}</strong>
+            ],
+            [
+              'Average Grade',
+              <strong>{roundToOneDecimcal(average(grades))}%</strong>
+            ],
+            [
+              'Median Grade',
```

Comment:
I did not notice this earlier, BE is sending the average grade already and can also send median. I can round the decimal to one as you are doing it here. This is how it is going to look. I think UI don't need to do the calculation can simply grab the value, what do you say?
```
[  
   {  
      "current_grade":95.9,
      "current_user_grade":null,
      "graph_upper_limit":110,
      "tot_students":35,
      "grade_avg":95.0,
      "grade_median":95.9
   }
]
```

Reply:
This is a simple change at the BE
```
 average_grade = df['current_grade'].mean().round(1)
 median_grade = df['current_grade'].median().round(1)
 df['grade_avg'] = average_grade
 df['grade_median']=median_grade
```

Reply:
I'm happy with backend doing calculations, however right now for the histogram we need the actual grades sent as well, so it's easy enough to calculate average and median. When we address #542 then we won't be sending all grades, so the backend calculation will be needed.

Reply:
ok.